ucm2/platforms/msm8916/qdsp6-components.conf

```diff
@@ -1,11 +1,21 @@
 If.qdsp6 {
-	Condition {
-		Type String
-		Haystack "${CardComponents}"
-		Needle "qdsp6"
-	}
-
-	True {
-		<platforms/msm8916/qdsp6.conf>
-	}
+
+        Condition {
```

Comment:
I guess I'd prefer if this stays as tabs instead of spaces

Reply:
This is what I get for copypasting from terminal instead of doing properly.